HolyshiftService/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.EntityFrameworkCore;
+using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
+using System;
+using System.Data.Common;
+using Microsoft.Extensions.Options;
+
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.Configure<HolyShiftDb>(builder.Configuration.GetSection("HolyShiftDb"));
+
+builder.Services.AddDbContext<HolyShiftDbContext>(options =>

```

Comment:
This is how I configure my DB usually which is a lot cleaner and concise
```
// Program.cs
builder.Services.Configure<DatabaseConfig>(builder.Configuration.GetSection("database"));
builder.Services.AddDbContext<PikaDataContext>();

// DBContext.cs
private readonly DatabaseConfig config;

public PikaDataContext(IOptions<DatabaseConfig> config)
{
    this.config = config.Value;
}

protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
{
  // ...
  optionsBuilder.UseMySql(...);
}
```

If you are wondering how the DBContext gets it's `DatabaseConfig` read about dependency injection. And C# does a huge part of it automatically for you so the above code is more than enough.